docs/development/rolling_update.md

```diff
@@ -2,6 +2,8 @@
 
 Upgrading and modifying a k8s cluster often requires the replacement of nodes.  To not cause loss of service and other disruptions a kops uses a functionality call rolling updates.  Rolling a cluster is the replacement of the masters and nodes with new cloud instances.
 
+Before rolling update, kops will check whether the instance in the ig need to be update, so when you just update nodes, the master will not roll, when you rolling be interrupted, you run again, will not roll the new instance.
```

Comment:
Thanks for improving the documentation, maybe this addition can be phrased a bit differently:

```suggestion
When starting the rolling update, kops will check each instance in the instance group if it needs to be updated, so when you just update nodes, the master will not be updated. When your rolling update is interrupted and you run another rolling update, instances that have been updated before will not be updated again.
```